csp.ipynb

```diff
@@ -1295,15 +1279,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now we are ready to create an InstruCSP instance for our problem. We are doing this for an instance of **MapColoringProblem** class which inherits from the **CSP** Class. This means that our **make_instru** function will work perfectly for it."
+    "Now we are ready to create an InstruCSP instance for our problem. We are doing this for an instance of **MapColoringProblem** class which inherits from the **CSP** Class. This means that our **make_instru** function should work perfectly for it."
```

Comment:
'should' adds a false sense of uncertainty. I think 'will' was fine but it's up to you.